### PR TITLE
Issue13

### PR DESCRIPTION
**Pull request by chunyuma**
This merge involves the following changes:
1. Resolved issue #7 and #13
2. Modified a few lines in the `make_training_data_from_sketches.py` to make it run faster
3. Modified the READM.md file to add the description of how to run the new function.

### CODE REVIEW

**Comment by Maximilian10** [comment]
@chunyuma could you please merge latest main into this branch?

**Comment by chunyuma** [comment]
ok @mfl15. The "main" branch is merged into this branch.

**Review by mfl15 — APPROVED** [review]

File: srcs/utils.py [hunk 1]
```diff
@@ -235,3 +216,176 @@ def write_processed_indices(filename, signatures, uncorr_org_idx):
...
+            else:
+                prediction.taxpathsn = None
+            profile.append(prediction)
+
+    # store profile for last sample
+    if 'SAMPLEID' in header and 'VERSION' in header and 'RANKS' in header:
+        if reading_data and len(profile) > 0:
+            samples_list.append((header['SAMPLEID'], header, profile))
+    else:
+        logger.error("Header in file {} is incomplete. Check if the header of each sample contains at least SAMPLEID, VERSION, and RANKS.\n".format(file_path))
```
Comment (mfl15) [inline srcs/utils.py:388]: image
@chunyuma could you please fix? Same 3 times in this file

Comment (chunyuma) [reply srcs/utils.py:388]: Thanks @mfl15. I fixed it https://github.com/KoslickiLab/YACHT/commit/3a06a77de9d3675edf18cf5aa8cd26871b2194aa

**Pull request merged**